### PR TITLE
fix: remove sushi and gnome photos from yafti

### LINKED ISSUE #595: [BUG] Issue with org.gnome.Photos and org.gnome.NautilusPreviewer not present on flathub-verified
**Describe the bug**
org.gnome.Photos not present on flathub-verified and org.gnome.NautilusPreviewer not present in flathub or flathub-verified
**To Reproduce**
1. run yafti
2. select photos and sushi from gnome core apps
3. see error
**Expected behavior**
Apps install or app options aren't present

**Actual behavior**
flatpak installation fails

**Your current image**
 Custom image based on secureblue userns silverblue image
```
State: idle
warning: Failed to query journal: couldn't find current boot in journal
AutomaticUpdates: stage; rpm-ostreed-automatic.timer: last run 23min ago
Deployments:
  ostree-image-signed:docker://ghcr.io/radioaddition/secureblue-custom-gnome:latest
                   Digest: sha256:7d8aa93b53c180aa89178092861dc25852cdd26ff984c952c067bfee646e8ef2
                  Version: 41.20241116.0 (2024-11-17T06:33:00Z)

● ostree-image-signed:docker://ghcr.io/radioaddition/secureblue-custom-gnome:latest
                   Digest: sha256:2833acde80f31f07968db22a0825319f74e709e0b715dcc8082b49f8016b2c16
                  Version: 41.20241116.0 (2024-11-17T05:01:14Z)

  ostree-image-signed:docker://ghcr.io/radioaddition/secureblue-custom-gnome:latest
                   Digest: sha256:f4488f3f848b84c3f401633619ff36a356e1198b6ff857828870c9906b4c7db6
                  Version: 41.20241116.0 (2024-11-16T21:20:12Z)
```

### PR DESCRIPTION
**Pull request by spaceoden**
[Sushi](https://flathub.org/apps/org.gnome.NautilusPreviewer) (the nautilus file previewer) and [Gnome Photos](https://flathub.org/apps/org.gnome.Photos) (a photo library app) are not flathub verified, so yafti should not offer to install them.

Fixes #595

### CODE REVIEW

File: recipes/common/silverblue-modules.yml [hunk 1]
```diff
@@ -6,6 +6,7 @@ modules:
         - adw-gtk3-theme
         - gnome-epub-thumbnailer
         - gnome-tweaks
+        - sushi
```
Comment (RoyalOughtness) [inline recipes/common/silverblue-modules.yml]: I agree with the removal of the two unverified flatpaks, but why is sushi needed? it seems like loupe serves this need

Comment (spaceoden) [reply recipes/common/silverblue-modules.yml]: It's not vital, but it's a feature of nautilus, so I figured for a vanilla Gnome experience it should be included. I've now removed it from the PR, so now this just fixes the unavailable flatpaks being requested.

**Review by RoyalOughtness — APPROVED** [review]

**Review by TheHooly — APPROVED** [review]

**Comment by jermanuts** [comment]
https://lemmy.world/post/27398324 wouldn't recommend Sushi even if it was verified.

**Comment by RoyalOughtness** [comment]
> https://lemmy.world/post/27398324 wouldn't recommend Sushi even if it was verified.

that's not related to the flatpak

**Pull request merged**